plugins/faustwp/includes/rest/callbacks.php

```diff
@@ -476,3 +488,42 @@ function handle_rest_telemetry_decision_callback( \WP_REST_Request $request ) {
 	);
 	return rest_ensure_response( $response );
 }
+
+/**
+ * Callback for WordPress register_rest_route() 'callback' parameter.
+ *
+ * Handle POST /faustwp/v1/validate_secret_key response.
+ *
+ * @link https://developer.wordpress.org/reference/functions/register_rest_route/
+ * @link https://developer.wordpress.org/rest-api/extending-the-rest-api/routes-and-endpoints/#endpoint-callback
+ *
+ * @param \WP_REST_Request $request Current \WP_REST_Request object.
+ *
+ * @return mixed A \WP_REST_Response, or \WP_Error.
+ */
+function handle_rest_validate_secret_key_callback( \WP_REST_Request $request ) {
+	return new \WP_REST_Response(
```

Comment:
I tried making a suggestion here to make it easy to commit the change, but GitHub keeps messing it up and doing partial selections/suggestions.

This should work to remove the `sprintf` call:
```
	return new \WP_REST_Response(
		esc_html__( 'Secret key validated!', 'faustwp' ),
		200
	);
```